packages/@expo/fingerprint/src/hash/Hash.ts

```diff
@@ -64,6 +65,18 @@ export async function createFingerprintSourceAsync(
   let result: HashResult | null = null;
   switch (source.type) {
     case 'contents':
+      if (typeof options.fileHookTransform === 'function') {
```

Comment:
assuming the types are accurate, it should be fine to just check non-nullish
```suggestion
      if (options.fileHookTransform) {
```